AppCenterData/AppCenterData/MSData.m

```diff
@@ -823,6 +826,40 @@ - (void)authTokenContext:(MSAuthTokenContext *)__unused authTokenContext didUpda
   }
 }
 
+#pragma mark - MSServiceNotificationDelegate
+
+- (void)appCenter:(MSAppCenter *)__unused appCenter didReceiveServiceNotification:(NSDictionary<NSString *, NSString *> *)notificationData {
```

Comment:
This is how the delegation design works. You can think of it as the "event" emitter. I like to think that it's also good to avoid method signature conflicts in whatever class it's adopted.